Card component and structure

joncodeofficial opened the pull request.

xanderazuaje committed:
Merge pull request #12 from xanderazuaje/dev

Dev

joncodeofficial committed:
fix: :rocket: Eliminando todo lo relacionado a js para tener todo en ts

joncodeofficial committed:
chore: :zap: Se paso al formato webp la imagen del formulario.

Ahora la imagen es mas ligera, mantiene  calidad y carga mas rapido

joncodeofficial committed:
chore: :rocket: se agrego logo de whatsapp

joncodeofficial committed:
refactor: :art: Se creo el componente Card.tsx para la card de contactos

xanderazuaje commented on src/components/UI/Card.tsx:
```diff
@@ -0,0 +1,106 @@
+              onClick={() => setIsExpanded(!isExpanded)}
+              className="text-[#D00B27] mx-auto cursor-pointer"
+            >
+              {isExpanded ? "Ver menos" : "Ver más"}
```
Cuando se clica el «Ver más» mueve también la posición de los elementos que estén en la misma fila. Sugiero alinearlos todos hacia arriba

joncodeofficial committed:
fix: :bug: Solucionado problema a la hora de abrir el accordion. movia a los elementos de la misma fila

Habia un problema a la hora de abrir la descripcion en ver mas. se movian los elementos de las misma fila

joncodeofficial committed:
fix: :rocket: Parte del problema del componente Card.tsx venia de aqui
